### ISSUE TITLE
Inconsistency for fetch_markets between normal ccxt and async

### ISSUE THREAD

**Issue by Stock84-dev** [ccxt/ccxt#3695]
- OS: Windows 7
- Programming Language version: python 3.7
- CCXT version: 1.17.154.
- Exchange: anxpro
- Method: fetch_markets

In normal ccxt all objects that are returned are lists while with async support some are list and some are dictionary. This happens with a lot of exchanges.

**Comment by kroitor** [comment]
@Stock84-dev can you make the shortest possible snippet (but a complete one, including the instantiation) to reproduce the issue?

**Comment by Stock84-dev** [comment]
```
import ccxt.async_support as ccxt_async
import ccxt
import asyncio

print(asyncio.get_event_loop().run_until_complete(ccxt_async.anxpro().fetch_markets()))
print(ccxt.anxpro().fetch_markets())

```
Output:
> {'BTC/USD': {'fee_loaded': False, 'percentage': True, 'maker': 0.003, 'taker': 0.006, 'precision': {}, 'limits': {}, 'id': 'BTCUSD', 'symbol': 'BTC/USD', 'base': 'BTC', 'quote': 'USD', 'multiplier': 100000}, 'BTC/HKD': {'fee_loaded': False, 'percentage': True, 'maker': 0.003, 'taker': 0.006, 'precision': {}, 'limits': {}, 'id': 'BTCHKD', 'symbol': 'BTC/HKD', 'base': 'BTC', 'quote': 'HKD', 'multiplier': 100000}...

> [{'fee_loaded': False, 'percentage': True, 'maker': 0.003, 'taker': 0.006, 'precision': {}, 'limits': {}, 'id': 'BTCUSD', 'symbol': 'BTC/USD', 'base': 'BTC', 'quote': 'USD', 'multiplier': 100000}, {'fee_loaded': False, 'percentage': True, 'maker': 0.003, 'taker': 0.006, 'precision': {}, 'limits': {}, 'id': 'BTCHKD', 'symbol': 'BTC/HKD', 'base': 'BTC', 'quote': 'HKD', 'multiplier': 100000}...

**Comment by kroitor** [comment]
Fixed in 1.17.156. Thx for reporting!
